api/GlobalEventHandlers.json

```diff
@@ -3891,10 +3891,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "1.3"
```

Comment:
Was `onselectionchange` perhaps supported earlier on some other interface?

Reply:
Nope, no other interfaces supported it when testing in Safari 4, so I'm not sure where the "1.3" came from.